### PR TITLE
docs(adapter): expresly specified what is supported

### PR DESCRIPTION
**Pull request by fzn0x**

### CODE REVIEW

**Comment by fzn0x** [comment]
This changes will closes [#2117](https://github.com/honojs/hono/issues/2117)

**Comment by yusukebe** [comment]
@fzn0x 

Thanks!

File: helpers/adapter.md [hunk 1]
```diff
@@ -20,26 +20,40 @@ import { env, getRuntimeKey } from 'https://deno.land/x/hono/helper.ts'
...
-Supported Runtimes:
+Supported Runtimes, Serverless Platforms and Cloud Services:
 
 - Cloudflare Workers
+  -  wrangler.toml
 - Deno
+  -  `Deno.env.set`
+  -  `.env` file
 - Bun
+  -  `process.env`
```
Comment (yusukebe) [inline helpers/adapter.md:44]: Bun supports also `Bun.env`.
https://bun.sh/guides/runtime/set-env

File: helpers/adapter.md [hunk 2]
```diff
@@ -20,26 +20,40 @@ import { env, getRuntimeKey } from 'https://deno.land/x/hono/helper.ts'
...
 })
 ```
 
-Supported Runtimes:
+Supported Runtimes, Serverless Platforms and Cloud Services:
 
 - Cloudflare Workers
+  -  wrangler.toml
 - Deno
+  -  `Deno.env.set`
```
Comment (yusukebe) [inline helpers/adapter.md:41]: How about just writing `Deno.env` and linking to this page? https://docs.deno.com/runtime/manual/basics/env_variables

File: helpers/adapter.md [hunk 3]
```diff
@@ -20,26 +20,40 @@ import { env, getRuntimeKey } from 'https://deno.land/x/hono/helper.ts'
...
   const { NAME } = env<{ NAME: string }>(c)
   return c.text(NAME)
 })
 ```
 
-Supported Runtimes:
+Supported Runtimes, Serverless Platforms and Cloud Services:
 
 - Cloudflare Workers
+  -  wrangler.toml
```
Comment (yusukebe) [inline helpers/adapter.md:39]: Please write with code span:

```md
`wrangler.toml`
```

File: helpers/adapter.md (hunk 1 above)
Comment (fzn0x) [reply helpers/adapter.md:45]: Resolved ✅

File: helpers/adapter.md (hunk 3 above)
Comment (fzn0x) [reply helpers/adapter.md]: Resolved ✅

**Comment by fzn0x** [comment]
I have completed addressing all the feedback provided. @yusukebe 👍

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@fzn0x 

Thanks!

**Pull request merged**